jaxrs-api/src/main/java/javax/ws/rs/core/AbstractMultivaluedMap.java

```diff
@@ -30,7 +31,7 @@
  * @param <V> the type of mapped values.
  * @author Marek Potociar
  */
-public abstract class AbstractMultivaluedMap<K, V> implements MultivaluedMap<K, V> {
+public abstract class AbstractMultivaluedMap<K, V> implements MultivaluedMap<K, V>, Serializable {
```

Comment:
@chkal I am uncertain whether adding the `Serializable` _marker_ interface could imply any backwards compatibility issues with existing third-party code: _In theory_ somebody could have extended this abstract class so his code could fail now due to missing implementation of the methods of `Serializable`, on the other hand `Serializable` is a pure marker interface with *no* methods at all. I personally think this is too risky for `2.1.1-SNAPSHOT` and we should move it to `2.2-SNAPSHOT` instead? What do you think?

Reply:
That's a really good question. I agree that adding it to `2.1.1-SNAPSHOT` could be too risky. But I'm not very familiar with the technical details of serialization to give a qualified answer here.

Reply:
Yes, this is 2.2-SNAPSHOT.

Reply:
I have updated the PR to 2.2-SNAPSHOT.